src/App.js

```diff
@@ -1,24 +1,13 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import BookList from "./containers/BookList";
+import BookForm from "./containers/BookForm";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <BookList />
+      <BookForm />
```

Comment:
i know this code works, but per requirement, please place this `APP`  component inside ` src/components/` folder 
![Screen Shot 2019-12-19 at 2 46 56 PM](https://user-images.githubusercontent.com/40712459/71177789-0b9f7280-226d-11ea-9f82-abe538bd4137.png)